### DISCUSSION TITLE
NEWS FLASH... PEASE READ!!!!

### DISCUSSION THREAD

**Discussion by kdschlosser** [lvgl-micropython/lvgl_micropython#69] [Announcements]
06/29/2004 12:55AM 

My ability to do any programming is going to be placed on hold for a few weeks. I was working on my car this morning finishing up a timing chain replacement and while I was under the car getting the power steering pump back into place the engine slid on the jack and crushed my right hand. 25 stitches to kind of put back together my middle finger and reattach part of it to my palm. This has resulted in tendons no longer being connected and nerves being severed, so not able to move the finger anymore. The nerve being cut causes the body to interpret the lack of signal from the nerve as the finger no longer being there and that gets processed as a damned lot of pain.

I do have photos if anyone is wanting to see what it looks like. It's nauseating to see but for some damned reason it is interesting to look at.

Moving the mouse and clicking the buttons with my left hand is not all that easy to do. As it turns out a persons brain actually wants to do everything as an exact mirror of what the right hand did. I have to put some crazy amount of concentration into moving the mouse to move it in the right direction.

I will do as much as I am able to. Discomfort being one of the things that is going to limit my ability but the thing that is going to make the biggest impact is frustration. The frustration is going to come from having to learn how to use the mouse with my left hand and eventually learning to type with a middle finger that has zero feeling and  is not going to move properly if at all. The expectation is that the finger is probably not going to move when my brain tells it to.

I don't give up so this is only going to slow down development. It is not going to end it.

There are a few requests I have for you guys/gals.
* Please, Please, Please, be patient. I may or may not come across as being bothered or annoyed, please do not take it personally. I assure you this will be frustration with myself and it is spilling over. 
* If I get frustrated and it can be seen in my responses to people make sure to put me in my place, let me know if you think I am being an ass in some manner. The best help I could get is to be reminded of the things I have said in this discussion. 
* Push me to keep working. I know I didn't loose an arm or anything like that for sure. This injury is going to make me have to relearn how to do a lot of things. It's not only my middle finger that is effected, everything in a persons hand is a very complex machine and if one tooth on one gear in that machine breaks it messes up the entire machine.

Thanks for your patience and understanding.

--Kevin Schlosser

Copied first 1/2 of this message from a response to an issue. I finished writing the second 1/2 and editing it to fix typos and other issues at 1:20AM. 25 minutes it has taken me to write 1/2 of this.

injury happened at ~7:00AM on 06/28/2024

**Comment by ikkesia** [comment]
I am sorry about this bad accident. I wish you the best recovery.
I hope to have you ready soon with your skill and availability.

**Comment by GC-RnD** [comment]
Kevin I'm sorry to hear this...
but you take all the time you need, your a workaholic anyway... 
take this 4th holiday, enjoy it with your family and friends and relax !!!
When the time is right you'll be back.

**Reply by kdschlosser** [reply]
Family is all on the east coast and my wife went to go visit her sister and cousin in Utah. It is only me.... well I have my 5 English Bulldogs to keep me company..

**Comment by ste7anste7an** [comment]
Hi Kevin, sorry to hear that you had this bad accident. Hopefully you recover well. Compared to this injury, the work on this lvgl library should be the least of your concerns. So, take all the time you need to recover.

**Comment by tairabiteru** [comment]
So sorry to hear about this. Please take all the time you need, and rest well! My silly little smart watch project can wait, there are more important things in this world. :)

Godspeed,
-Taira

**Comment by kdschlosser** [comment]
thanks guys. I know this puts the brakes on this project and as a result yours is also going to be put on hold.  Thank you for being very understanding. once I get past the next few days and the pain subsides and the finger splint comes off I will se how things go from there. I have to say 2024 so far has been a pretty crappy year.

Wife had to have a toe amputated.
Wife was diagnosed with melanoma and she had surgery to remove that.
Wife's father had surgery to fix 2 hernias.
Wife's father found out he has prostate cancer.
Wife's father's wife fell and broke her hip.
My mom had knee surgery. 
My mom had heart surgery.
My mom fell and broke her hip.
I had a back surgery.
and that wasn't enough I guess and I had to end up with the crushed hand.

I am really hoping that the rest of the year goes by without indecent. That is a long list for 6 months of time. 
There is a saying I came up with a long time ago... If life wasn't full of adversity, it'd be boring. But this is getting to be a little bit too much... I am waiting for something to happen to offset all of the crazy this year has brought so far.

**Reply by bdbarnett** [reply]
Hey friend,

I am very sorry to hear this. Hang in there! We're all rooting for you!

**Reply by kdschlosser** [reply]
It is what it is m8. Gotta roll with the punches I guess.

**Comment by kdschlosser** [comment]
Good news is I am already using the finger to type and I am using my right hand to move the mouse.... so I'm healing pretty fast. Lots of typos because I don't have feeling in my middle finger and I am not pressing the keys down all the way. Gonna have to work on that one.

**Comment by ikkesia** [comment]
Happy to hear this good news !

**Comment by Mustigusti** [comment]
I wish you a speedy recovery Kevin! Take all the time you need you really deserve it.

**Comment by kdschlosser** [comment]
I am actually healing really fast. Typing is doing OK and I am able to use the mouse with my right hand. I am shocked at how fast the healing process is going.
